### PR TITLE
Dstress

### PR DESCRIPTION
**Pull request by joshskkim**
@RetailHub/gandalf Initial proxy with only one service turned on

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -24,3 +24,4 @@ coverage
 build/
 dist/
 
+config.js
```
Comment (sajjanrajvaidya) [inline .gitignore:27]: Why ignore this file?

File: public/index.html [hunk 2]
```diff
@@ -7,13 +7,13 @@
...
-    <div id="reviewsapp"></div>
-    <script  src='http://ec2-3-132-5-204.us-east-2.compute.amazonaws.com:3001/singleItemPage.js'></script>
-    <script  src='http://34.201.53.74:3002/dist/bundle.js'></script>
+    <!-- <div id="sponsored-related-products"></div>
+    <div id="reviewsapp"></div> -->
+    <script  src='http://localhost:3001/singleItemPage.js'></script>
+    <!-- <script  src='http://34.201.53.74:3002/dist/bundle.js'></script>
      <script  src='http://54.89.234.183:3003/bundle.js'></script>
-    <script  src='http://18.212.184.37:3004/bundle.js'></script>
+    <script  src='http://18.212.184.37:3004/bundle.js'></script> -->
```
Comment (sajjanrajvaidya) [inline public/index.html:17]: What is happening here? I thought you were meant to comment out all other services for now.

File: .gitignore (hunk 1 above)
Comment (joshskkim) [reply .gitignore:27]: its a key! be safe on the web

File: public/index.html (hunk 2 above)
Comment (joshskkim) [reply public/index.html:17]: I did

**Pull request merged**